webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/activity/footer/actions/ActivityShareAction.vue

```diff
@@ -68,6 +69,9 @@ export default {
     shareIconColorClass() {
       return this.hasShared && 'primary--text' || 'disabled--text';
     },
+    disableButtons() {
```

Comment:
use simple word to define the whole component as disabled

```suggestion
    disabled() {
```